### PR TITLE
Display AbsoluteOrientationSensor.quaternion elements to four rows

### PR DESCRIPTION
**Pull request by xiuqijix**
PNP team need use this demo to test latency, so let the each value one row is better for comparison.

### CODE REVIEW

File: src/sensors-app.html [hunk 1]
```diff
@@ -206,15 +206,20 @@
...
                         } else if (propertyName in this) {
-                            var readingId = 'readingValue' + i++;
-                            var value = null;
-                            if (!isNaN(this[propertyName])) {
-                                value = round(this[propertyName], 3);
+                            if (propertyName == 'quaternion') {
+                              this["readingValue0"] = propertyName + ".X: " + round(this[propertyName][0], 3);
+                              this["readingValue1"] = propertyName + ".Y: " + round(this[propertyName][1], 3);
+                              this["readingValue2"] = propertyName + ".Z: " + round(this[propertyName][2], 3);
+                              this["readingValue3"] = propertyName + ".W: " + round(this[propertyName][3], 3);
```
Comment (alexshalamov) [inline src/sensors-app.html:214]: This template placeholder is missing, you should add it before you can use it. (line 83)
https://v-two-dot-genericsensors.appspot.com/ without new placeholder, W would not be shown.

**Review by alexshalamov — COMMENTED** [review]
Please add 1 template placeholder

File: src/sensors-app.html (hunk 1 above)
Comment (xiuqijix) [reply src/sensors-app.html:214]: Add `<span>[[item.readingValue3]]</span>`

**Review by alexshalamov — APPROVED** [review]

**Comment by alexshalamov** [comment]
lgtm, new version pushed to https://genericsensors.appspot.com/

**Pull request merged**